packages/babel-plugin-transform-for-of/src/index.js

```diff
@@ -6,14 +6,20 @@ import transformWithoutHelper from "./no-helper-implementation";
 export default declare((api, options) => {
   api.assertVersion(7);
 
-  const { loose, assumeArray } = options;
+  const { loose, assumeArray, allowArrayLike } = options;
```

Comment:
Might worthy to note that this option requires `babel/core @ 7.9.0`, otherwise it is an no-op.